gapic_smoketest.py

```diff
@@ -29,85 +29,50 @@
-    "java" : lambda api_name, api_version, log : runJavaTests(api_name, api_version, log),
-    #TODO: all other languages
-}
-
 test_apis = {
     "pubsub" : ("v1", "google/pubsub/artman_pubsub.yaml"),
     "logging" : ("v2", "google/logging/artman_logging.yaml"),
     "speech" : ("v1", "google/cloud/speech/artman_speech_v1.yaml"),
 }
 
-def runJavaTests(api_name, api_version, log):
-    gapic_dir = "gapic-google-cloud-%s-%s" % (api_name, api_version)
-    # Run gradle test in the main java directory and also in the gapic directory.
-    cmds = ("%s/gradlew build test" % os.getcwd()).split()
-    exit_code = subprocess.call(cmds, stdout=log, stderr=log)
-    if exit_code:
-        return exit_code
-
-    cmds = ("%s/gradlew -p %s build test" % (os.getcwd(), gapic_dir)).split()
-    return subprocess.call(cmds, stdout=log, stderr=log)
 
-def run_smoke_test(api_name, language, root_dir, log, user_config):
+def generate_clients(language, root_dir, log, user_config):
     log_file = _setup_logger(log)
     failure = []
     success = []
     warning = []
-    (api_version, artman_yaml_path) = test_apis[api_name]
-    target = language + "_gapic"
-
-    # Generate client library for an API and language.
-    if _generate_artifact(artman_yaml_path,
-                          target,
-                          root_dir,
-                          log_file,
-                          user_config):
-        msg = 'Failed to generate %s of %s.' % (
-            target, artman_yaml_path)
-        failure.append(msg)
-        logger.info(msg)
-    else:
-        msg = 'Succeded to generate %s of %s.' % (
-            target, artman_yaml_path)
-        success.append(msg)
-        logger.info(msg)
-
-        # Test the generated client library.
-        logger.info("Starting testing of %s %s client library." %(language, api_name))
-        cwd = os.getcwd()
-        os.chdir("artman-genfiles/%s" % language)
-
-        if _test_artifact(test_languages[language], api_name, api_version, log_file):
-            msg = 'Tests failed for %s library.' % (
-                language)
+    for api_name in test_apis:
+        (api_version, artman_yaml_path) = test_apis[api_name]
+        target = language + "_gapic"
+        # Generate client library for an API and language.
+        if _generate_artifact(artman_yaml_path,
+                              target,
+                              root_dir,
+                              log_file,
+                              user_config):
+            msg = 'Failed to generate %s of %s.' % (
+                target, artman_yaml_path)
             failure.append(msg)
         else:
-            msg = 'Tests passed for %s library.' % (
-                language)
+            msg = 'Succeded to generate %s of %s.' % (
+                target, artman_yaml_path)
             success.append(msg)
-        os.chdir(cwd)
         logger.info(msg)
     logger.info('================ Smoketest summary ================')
```

Comment:
Does `Smoketest` still apply here? It seems like this is just now generation messages.

Reply:
Done.